### PR TITLE
js_task-transportation-on-vacation PR

### PR DESCRIPTION
**Pull request by Rocketjet**

### CODE REVIEW

**Review by roman-toritsyn — APPROVED** [review]

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -25,10 +25,18 @@
...
  */
-
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days < 7) {
+    return days * 40 - 20;
```
Comment (Ihor-Karpyn) [inline src/calculateRentalCost.js:36]: don't repeat yourself `days * 40`

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -25,10 +25,18 @@
...
+  } else {
+    return days * 40 - 50;
```
Comment (Ihor-Karpyn) [inline src/calculateRentalCost.js:38]: don't repeat yourself  `days * 40`

File: src/calculateRentalCost.js [hunk 3]
```diff
@@ -25,10 +25,18 @@
...
-
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days < 7) {
+    return days * 40 - 20;
+  } else {
```
Comment (Ihor-Karpyn) [inline src/calculateRentalCost.js:37]:
```suggestion
  }
```

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]
Almost done

**Review by Anna-Tkachenko — APPROVED** [review]

**Pull request closed without merging**